static/src/javascripts/projects/common/modules/experiments/__fixtures__/ab-test.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+import type { ABTest, Variant } from '../ab-types';
+
+export const genVariant = (id: string): Variant => ({
+    id,
+    test: () => undefined,
+});
+
+export const genAbTest = (id: string, canRun: ?boolean): ABTest => ({
```

Comment:
Thinking about this now I think this may be overkill, as none of the tests currently need it (and have been that way for some time). 
If we add new tests in the future and there's a demand for configuring more of the test options I think this would be worthwhile, but since nothing would actually need it now, I'm not sure there's a reason to change it